### PR TITLE
Compressed unzip wizard GIF animation

### LINKED ISSUE #36711: Unzip wizard GIF animation too heavy
Following this [blog post](https://build.prestashop-project.org/news/2023/new-project-branding/), the prestashop project moves on.

This change was be implemented by 3 PRs:
1. https://github.com/PrestaShop/prestashop-ui-kit/pull/229
2. https://github.com/PrestaShop/PrestaShop/pull/36584
3. https://github.com/PrestaShop/PrestaShop/pull/36585

See https://github.com/PrestaShop/PrestaShop/issues/36615

In PR https://github.com/PrestaShop/PrestaShop/pull/36585 the image used is very big, which makes the downloadable Zip very big. The image used must be optimized.

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Compressed version of the GIF for the installer
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Build a .zip and try to install it manually and check if the new GIF is ok
| UI Tests          | 
| Fixed issue or discussion?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/36711
| Related PRs       | 
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by jolelievre — APPROVED** [review]
Thanks @tblivet

**Review by matks — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Comment by matks** [comment]
Thank you @tblivet

**Pull request merged**
